src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
```

Comment:
@jajreidy @lslebodn I believe we can get the `image_name`  from existing attributes just using something similar to [this](https://github.com/release-engineering/pubtools-ami/blob/main/pubtools/_ami/tasks/push.py#L140-L173) in our _pubtool_, no?

Reply:
That'd probably be fine.